js/site-search.js

```diff
@@ -62,7 +77,7 @@ bcpl.siteSearch = (($, window, constants) => {
 	$(document).on('click', siteSearchSearchIconSelector, onSearchIconClick);
 	$(document).on('click', searchButtonCatalogSelector, onSearchCatalogClick);
 	$(document).on('keyup', siteSearchInputSelector, onSiteSearchKeyup);
-	// $(document).on('click', searchButtonEventsSelector, onSearchEventsClick);
+	$(document).on('click', searchButtonEventsSelector, onSearchEventsClick);
 	// $(document).on('click', searchButtonWebsiteSelector, onSearchWebsiteClick);
```

Comment:
remove?

Reply:
I'm going to leave it since it's getting used in another branch that will succeed this one.

Reply:
can you just thrown in a comment saying that?